### PR TITLE
docs(ui): discussion around ui@5 changes for forms

### PR DESCRIPTION
**Pull request by varl**

### CODE REVIEW

File: decisions/2020/04/28-ui-forms-refactor [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Agenda
+
+Simplification and consistency of the form controls, e.g. Radio, Checkbox, Switch, Field, *Field-components.
+
+Touched on future changes we want to explore for ui@6 as well.
+
+# @dhis2/ui-core
+
+- Drop the Field component
+- Re-implement as a generic form field
```
Comment (ismay) [inline decisions/2020/04/28-ui-forms-refactor]: So to clarify for myself, what we mean here is basically; change the `<Field>` component from just something that provides spacing, to a component that wraps a form 'field' (for lack of a better word) and adds helpText, validationText, etc. Is that right?

**Comment by Mohammer5** [comment]
> - Use more explicit style for controls instead of e.g. `options`.

I don't remember what this means, could you explain it?

<hr />

Another thing I remember is that we've decided to remove margins on container elements in our base/core components

File: decisions/2020/04/28-ui-forms-refactor (hunk 1 above)
Comment (ismay) [reply decisions/2020/04/28-ui-forms-refactor]: So that you can do:

```jsx
<Field helpText="Help text" {// etc.}>
  <Input {...} />
</Field>
```

Comment (varl) [reply decisions/2020/04/28-ui-forms-refactor]: That's correct: https://github.com/dhis2/ui/pull/46/files#diff-0556b6ad0bddb321a9ddd41f13279cb6R31-R48

**Comment by varl** [comment]
> Another thing I remember is that we've decided to remove margins on container elements in our base/core components

I will add this under "future" and we can determine if this is a v5 thing or v6 thing. If it's only form components, maybe v5 is viable. If we do it on a lot of components, maybe v6 is better.

**Comment by varl** [comment]
> > * Use more explicit style for controls instead of e.g. `options`.
> 
> I don't remember what this means, could you explain it?

Will do !

File: decisions/2020/04/28-ui-forms-refactor (hunk 1 above)
Comment (ismay) [reply decisions/2020/04/28-ui-forms-refactor]: Ok cool, thx 👍

**Comment by varl** [comment]
Does acb94c5 address your concerns? Thumbs up this message if so and I'll proceed.

**Pull request merged**